tcms/testruns/tests/test_admin.py

```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=invalid-name
 from django.urls import reverse
-from tcms.tests import LoggedInTestCase
+from tcms.tests import LoggedInTestCase, BaseCaseRun
 from tcms.utils.permissions import initiate_user_with_default_setups
+from tcms.tests.factories import TestRunFactory
 
 
-class TestTestRunAdmin(LoggedInTestCase):
+class TestTestRunAdmin(BaseCaseRun):
```

Comment:
Prefer creating the data used for testing in setUpData instead of introducing the entire inheritance chain here. Using factories like you probably meant to is also a 1 line change.